README.md

```diff
@@ -129,17 +129,8 @@ coroutine dispatcher and also makes sure that in case of crashed coroutine with
 exception is logged before crashing Android application, similarly to the way uncaught exceptions in 
 threads are handled by Android runtime. 
 
-### ProGuard
-
-In obfuscated code, fields with different types can have the same names,
-and `AtomicReferenceFieldUpdater` may be unable to find the correct ones.
-To avoid field overloading by type during obfuscation, add this to your config:
-
-```
--keepclassmembernames class kotlinx.** {
-    volatile <fields>;
-}
-```
+### R8 and ProGuard
+If you are using R8 or ProGuard add the options from [coroutines.pro](core/kotlinx-coroutines-core/resources/META-INF/proguard/coroutines.pro) file to your rules.
```

Comment:
Style note: let's put an empty line after titles (it reads better in source, since it is actually a separate paragraph).